Emittery.d.ts

```diff
@@ -76,6 +76,16 @@ declare class Emittery {
 	 * specified.
 	 */
 	listenerCount(eventName?: string): number;
+
+	/**
+	 * Bind a `methodNames` or all `Emittery` methods if `methodNames` not defined into `target` object.
+	 */
+	bindMethods(target: Object, methodNames?: string[]): void;
```

Comment:
```suggestion
	bindMethods(target: object, methodNames?: readonly string[]): void;
```

Reply:
For the record, readonly doesn't work there, as the code itself is in js.

Reply:
Oh, it doesn't even compile. `readonly` isn't allowed for function arguments.

Reply:
> readonly isn't allowed for function arguments.

It is: https://devblogs.microsoft.com/typescript/announcing-typescript-3-4/#improvements-for-readonlyarray-and-readonly-tuples

It doesn't actually have any purpose. It will just make it easier to move everything to TS later on.

Reply:
Oh, you're right, had older version and was looking at it.